pilot/cmd/pilot-agent/status/server.go

```diff
@@ -671,7 +687,22 @@ func (s *Server) handleQuit(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	_, _ = w.Write([]byte("OK"))
 	log.Infof("handling %s, notifying pilot-agent to exit", quitPath)
-	notifyExit()
+	s.shutdown()
```

Comment:
So this is ternimate immediately without draining, isnot this a compatibility change

Reply:
Good catch, I thought it quit immediately before. updated

Reply:
forget pushing commit

Reply:
sorry... done